test/unit/org/apache/cassandra/cql3/statements/DescribeStatementTest.java

```diff
@@ -462,11 +463,18 @@ public void testDescribeCluster() throws Throwable
 
         assertRowsNet(executeDescribeNet(KEYSPACE_PER_TEST, "DESCRIBE CLUSTER"),
                       row("Test Cluster",
-                          "ByteOrderedPartitioner",
+                          trimIfPresent(DatabaseDescriptor.getPartitionerName(), "org.apache.cassandra.dht."),
```

Comment:
`String.replaceFirst()`?

Reply:
Works as an implementation of `trimIfPresent` (`src.replaceFirst("^" + Pattern.quote(begin), "")`), but the current code is IMHO a little more obvious.